services/web/server/src/simcore_service_webserver/products/_rpc.py

```diff
@@ -24,6 +24,18 @@ async def get_credit_amount(
     return credit_result_get
 
 
+@router.expose()
+async def get_product_stripe_info(
```

Comment:
As I mentioned above, I am a bit surprised about this approach. I am not totally against it but we need to be careful not to open a door here that ends up out of control. In my experience this approach does not safely scale in complexity since it is not backed by good mechanisms for e.g. specs checking, validations, loops checking or error handling

Reply:
Yes, we can start to add additional validations. These are also the first small steps to get independent on Payment Gateway.